architektura-aplikacje.gov.pl.rst

```diff
@@ -0,0 +1,28 @@
+Architektura platformy aplikacje.gov.pl
+=======================================
+
+Niniejszy dokument przedstawia architekturę platformy aplikacje.gov.pl (w skład której wchodzi m.in. aplikacja EZD) od strony oprogramowania.
+
+Kluczowym elementem platformy aplikacyjnej będzie moduł obsługujący tożsamość użytkowników. Oprócz przechowywania kont użytkowników będzie on pozwalał na uwierzytelnianie ich za pomocą wymienialnych metod (np. hasło, podpis elektroniczny, cyfrowy dowód osobisty — eDowód). Moduł ten będzie również odpowiadał za wytwarzanie danych pozwalających na uwierzytelnienie w zewnętrznym API systemu (ciastka sesyjne lub alternatywne rozwiązania jak JSON Web Tokens).
+
+Na bazie modułu użytkowników będą tworzone aplikacje odpowiedzialne za grupy powiązanych ze sobą funkcjonalności. Aplikacje będą podzielone na moduły, korzystające po części ze wspólnego API programistycznego. Jednym z najważniejszych elementów interfejsu administracyjnego instancji aplikacje.gov.pl będzie więc komponent zarządzający dodawaniem nowych aplikacji oraz zależnościami pomiędzy nimi. Dodanie nowej aplikacji powinno być możliwe nawet dla użytkownika, który nie posiada zdolności programistycznych. Aplikacje będą we własnym zakresie kształtować polityki uprawnień określające dostęp do obiektów, którymi zarządzają.
+
+Wybrane obiekty w ramach poszczególnych aplikacji (uznawane za istotne z punktu widzenia prawnego) będą mogły być podpisywane (w postaci zserializowanej) z wykorzystaniem bądź to osobistego podpisu elektronicznego urzędnika (jeśli aplikacja kliencka udostępni taką możliwość), bądź pieczęci elektronicznej serwera. Do opakowania podpisów oraz znaczników czasu posłużą ustandaryzowane formaty kontenerów (CAdES/XAdES).
+
+Oprócz tego, poświadczanie utworzenia tych obiektów będzie dokonywane z wykorzystaniem Keyless Signature Infrastructure - dzięki tej technologii będzie można udowodnić, że pismo powstało w danym momencie, a nie np. zostało antydatowane przez złośliwego administratora.
```

Comment:
Usunąłbym "oprócz tego" - bo wnosi do tekstu informacji.

"Złośliwy administrator" mi nie pasuje. Złośliwy to (z sjp.pwn.pl) "lubiący sprawiać innym przykrość". Ten administrator przez antydatowanie nie chce sprawiać przykrości tylko uzyskać dla siebie jakieś korzyści. Zamieniłbym na "administratora o złych intencjach", albo na "administratora".

(Przepraszam, że przyczepiam się do takich mało merytorycznych rzeczy.)